gulpfile.js

```diff
@@ -110,6 +110,7 @@ function watch(done) {
   connect.server({
     https: argv.https,
     port: port,
+    host: argv.host,
```

Comment:
what happens if `host` is not passed in as an argument? would it default to the same value used today?

Reply:
You were right. I fixed it, so now it should have default value "localhost".